core/src/parking_lot.rs

```diff
@@ -16,6 +16,39 @@ use core::{
 use smallvec::SmallVec;
 use std::time::{Duration, Instant};
 
+cfg_if! {
+    if #[cfg(target_arch = "wasm32-unknown-unknown")] {
```

Comment:
That doesn't actually work, you need to check `target_os = "unknown"` and `target_vendor = "unknown"`.

Reply:
Oh ok, my bad.